examples/vision/deeplabv3_plus.py

```diff
@@ -0,0 +1,298 @@
+"""
+Title: Multiclass Semantic Segmentation using DeepLabV3+
+Author: [Soumik Rakshit](http://github.com/soumik12345)
+Date created: 2021/08/31
+Last modified: 2021/09/1
+Description: Implement DeepLabV3+ architecture for Multi-class Semantic Segmentation
+"""
+"""
+## Introduction
+"""
+
```

Comment:
You can remove the break between the two text blocks (it's just one block)

Reply:
Removed break between consecutive text blocks.